db/repos/transactions.multisignature.js

```diff
@@ -49,9 +52,11 @@ function MultiSigTransactionsRepo(db, pgp) {
 }
 
 /**
- * Save Multisignature transactions
+ * Save Multisignature transactions.
+ *
  * @param {Array.<{id: string, {asset: {multisignature: {min: int, lifetime: int, keysgroup: Array.<string>}}}>} transactions
```

Comment:
This typedef seems to break GitHub parsing. Is it valid?

Reply:
I found no example in the docs.
I simplified it by putting only {Array} as a type.
For the future we could use typedefs like recommended in the JSDoc documentation for describing more complex Objects and Arrays.